### DISCUSSION TITLE
Tortoise integeration

### DISCUSSION THREAD

**Discussion by SicariusSicariiStuff** [Open-LLM-VTuber/Open-LLM-VTuber#11] [Ideas]
Hi,

This is an awesome project! Integrating Tortoise would be fantastic.

I previously created an extension for Booga, but that project is now largely inactive. However, the Tortoise model finetuning is quite robust. It would be great if your project could support running Tortoise models.

For reference, here is my project repository:
[Diffusion_TTS](https://github.com/SicariusSicariiStuff/Diffusion_TTS)

And here are the model files:

[Diffusion_TTS](https://huggingface.co/SicariusSicariiStuff/Diffusion_TTS)

[TTS_Lola](https://huggingface.co/SicariusSicariiStuff/TTS_Lola)

Thank you!

**Comment by t41372** [comment]
Sorry for responding so late. I was on vacation for a while. I will look into it. Thank you for your suggestion.

**Comment by Space-Jelly** [comment]
hi，我想问一下Python3.12.6的版本可以吗？谢谢您的回答，打扰您了！！

**Comment by wqidnc** [comment]
incredible

**Comment by VED2032PATEL** [comment]
Fantastic

**Comment by rekkiem** [comment]
La tortuga me trae mucha nostalgia
